### PR TITLE
HCAP-1097 | BugFix | Show warning message on Unsuccessful cohort participant 

### PR DESCRIPTION
**Pull request by fwpushan**
----
- Showing warning message on Unsuccessful participant with continue No

Ticket: [HCAP-1097](https://freshworks.atlassian.net/browse/HCAP-1097)

### CODE REVIEW

File: client/src/components/modal-forms/ManageGraduationForm.js [hunk 1]
```diff
@@ -84,15 +86,17 @@ export const ManageGraduationForm = ({ initialValues, onClose, onSubmit, cohortE
...
-                    label='Withdraw this participant from the program.'
-                  />
-                )}
-
+                {values?.continue === 'continue_no' &&
+                  values.status === postHireStatuses.cohortUnsuccessful && (
+                    <Box>
+                      <MuiAlert severity='warning'>
+                        {
+                          'Participants who no longer wish to to continue in HCAP will need to be archived, please use the archive function provided after you submit.'
```
Comment (fwkendall) [inline client/src/components/modal-forms/ManageGraduationForm.js:94]: typo here: "to to"

File: client/src/components/participant-details/track-graduation.js [hunk 2]
```diff
@@ -100,7 +100,6 @@ export const TrackGraduation = (props) => {
                       : cohortEndDate,
                 },
                 continue: 'continue_yes',
-                withdraw: false,
```
Comment (fwkendall) [inline client/src/components/participant-details/track-graduation.js:103]: why was this removed? if they choose continue_yes, the participant should not be withdrawn from the program (only from the cohort)

Comment (fwpushan) [reply client/src/components/participant-details/track-graduation.js:103]: This is redundant info, we are tracking withdraw in different table.

File: client/src/components/modal-forms/ManageGraduationForm.js [hunk 3]
```diff
@@ -55,8 +56,8 @@ export const ManageGraduationForm = ({ initialValues, onClose, onSubmit, cohortE
                     name={'data.date'}
                     label={
                       values.status === postHireStatuses.postSecondaryEducationCompleted
-                        ? 'Graduation Date'
-                        : 'Unsuccessful Graduation Date'
+                        ? 'Graduation Date / Cohort End Date     '
```
[the remark this answers is not in the document]
Comment (fwpushan) [reply client/src/components/modal-forms/ManageGraduationForm.js:59]: Nothing specific in Design but there should be a indication that graduation date is pre-filled with Cohort-end date. What is your thought ?

File: client/src/components/participant-details/track-graduation.js (hunk 2 above)
Comment (fwpushan) [reply client/src/components/participant-details/track-graduation.js:103]: This variable is a form state-var to store previously implemented checkbox.

File: client/src/components/modal-forms/ManageGraduationForm.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (fwkendall) [reply client/src/components/modal-forms/ManageGraduationForm.js]: This is question for Tavy/Khanh but IMO no, I think the design text we have now is fine 😄

File: client/src/components/participant-details/track-graduation.js (hunk 2 above)
Comment (fwkendall) [reply client/src/components/participant-details/track-graduation.js:103]: Gotcha!

File: client/src/components/modal-forms/ManageGraduationForm.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (fwkendall) [reply client/src/components/modal-forms/ManageGraduationForm.js]: Oh I see you reverted, never mind! I missed the "outdated" label on this, resolving

**Review by fwkendall — APPROVED** [review]
